cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityOperator.java

```diff
@@ -62,7 +64,7 @@
      * @param labels
      */
     protected EntityOperator(String namespace, String cluster, Labels labels) {
-        super(namespace, cluster, labels);
+        super(namespace, cluster, labels, ENTITY_OPERATOR_NAME);
```

Comment:
I'm not sure this is correct. The Entity Operator is managed by the Cluster Operator. It doesn't manage it self. So I think everything created through the Abstract model should have the _managed by_ label set to something like `strimzi-cluster-operator`. I this applies to most of the files in the `cluster-operator` module. Do we really want/need to distinguish it beyond the process level? I think the process level (i.e. which pod is managing it might be more understandable for the user).